plugins/home/src/components/HomepageCompositionRoot.tsx

```diff
@@ -26,7 +31,9 @@ export const HomepageCompositionRoot = (props: {
   const children = props.children ?? outlet;
   return (
     <Page themeId="home">
-      <Header title={props.title ?? 'Home'} />
+      <Header title={props.title ?? 'Home'}>
+        <HomepageTimer />
```

Comment:
My personal opinion is that I think this is a component that could definitely come out of the box but I wonder if there are use cases where there is no need for the timer so that it should be possible to "compose" that as well? meaning that we should "lift out" the header component and content component to https://github.com/backstage/backstage/blob/master/packages/app/src/components/home/HomePage.tsx#L26 so that you can compose those too? What do you think @rubenLindstrom cc @backstage/techdocs-core

Reply:
Great insight! Yes, I think that makes sense! It was initially moved to reduce the amount of code needed when composing the homepage, and to have an area entirely dedicated to just composing what components you would like.

However, since a default home page will be generated by the CLI anyways, I think it's fine to have it in the app, to allow for more control for the app integrator! :)

An alternative solution would be to introduce a flag in the props to remove the clock now. However, I guess that would be harder to find, and given that the header can become more advanced in the future, perhaps it's best to pull the plug immediately and move it out.

Reply:
Please let me know @awanlin if you want to have help on this. Very much appreciate your contribution, and want to be sure it gets through :)

Reply:
How would I got about doing this? Are there any similar examples of this in code that I could look at? I'm happy to work on this change but I'm a little green with some of these concepts so help is much appreciated

Reply:
If I interpreted @emmaindal's suggestion correctly, to implement this we would remove the `Header` and the `Content` from the `HomePageCompositionRoot`, and instead include it to the example app, in `/packages/app/src/components/home/HomePage.tsx`. In the future, we should then also add it to the `create-app`, such that these are included by default when a new instantiation is created.

Does that make sense?

Reply:
Sounds good, will give that a try, thanks for the details